### PR TITLE
Add doi in versions

### PR DESCRIPTION
**Pull request by ines-cruz**
* add doi in the ui in the versions section

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js [hunk 1]
```diff
@@ -36,6 +37,11 @@ const RecordVersionItem = ({ item, activeVersion }) => {
             {item.publication_date}
           </p>
         </Grid.Column>
+        <Grid.Column width={12}>
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js:40]: In semantic-ui, columns are max `16`. I suspect you wanted to go to a new row here.
In that case, you have to:
```
</Grid.Row>
<Grid.Row columns={1}>
...
</Grid.Row>

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js [hunk 2]
```diff
@@ -36,6 +37,11 @@ const RecordVersionItem = ({ item, activeVersion }) => {
...
+          <p className="text-muted font-small">
+            {_.get(item.pids, "doi.identifier", "")}
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js:42]: I don't see the import of `get`: you should always import specific function whenever possible to take advantage of the [tree-shaking feature](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking).
```
import _get from 'lodash/get';
```

**Review by ntarocco — COMMENTED** [review]
Good job! It would be great if you add a screenshot to see the changes :)

**Comment by ines-cruz** [comment]
screenshot
In the versions section the doi is visible according to the version

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js (hunk 1 above)
Comment (ines-cruz) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js]: Thank you. I changed it in the other commit

**Comment by ppanero** [comment]
Styling-wise, basing on Zenodo, I think the DOI should be shown in the same `Row` component, but in a different line. See screenshot :)

Screenshot 2021-04-28 at 10 31 36

**Comment by ines-cruz** [comment]
test

**Review by ntarocco — APPROVED** [review]

**Pull request merged**